website/orders/api/serializers/orders.py

```diff
@@ -0,0 +1,131 @@
+from rest_framework import serializers
+from orders import models
+from orders.exceptions import OrderException
+from orders.services import add_order
+
+
+class ProductSerializer(serializers.ModelSerializer):
+    """Serializer for Product model."""
+
+    class Meta:
+        """Meta class."""
+
+        model = models.Product
+        fields = [
+            "id",
+            "name",
+            "icon",
+            "available",
+            "available_at",
+            "current_price",
+            "orderable",
+            "ignore_shift_restrictions",
+            "max_allowed_per_shift",
+            "barcode",
+        ]
+
+
+class OrderSerializer(serializers.ModelSerializer):
+    """Serializer for Order model."""
+
+    def create(self, validated_data):
+        """
+        Create an Order from validated serialized data.
+
+        This function uses add_order to add an Order to a Shift
+        :param validated_data: validated serialized data
+        :return: an Order if the Order could be successfully added, a ValidationError otherwise
+        """
+        try:
+            return add_order(
+                validated_data["product"],
+                validated_data["shift"],
+                validated_data["type"],
+                user=validated_data["user"],
+                paid=validated_data["paid"],
```

Comment:
shouldn't the API always create unpaid and unready orders, which an admin can then make paid and ready with a separate PATCH?